app/src/main/java/com/example/simplegithubapp/follow/FollowHolder.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.simplegithubapp.follow
+
+import androidx.recyclerview.widget.RecyclerView
+import com.example.simplegithubapp.databinding.RvItemFollowerBinding
+
+class FollowHolder(val binding:RvItemFollowerBinding): RecyclerView.ViewHolder(binding.root){
+    fun bind(data:RvFollowData){
+        binding.apply {
+            rvFollow=data
+        }
```

Comment:
```suggestion
    fun bind(data:RvFollowData){
        binding.rvFollow=data
    }
```
여기선 굳이 apply 안써도될듯?  쓸꺼면 한줄로 해도될듯